webapp/channels/src/components/menu/menu.tsx

```diff
@@ -176,7 +176,8 @@ export function Menu(props: Props) {
                 aria-label={props.menuButton?.['aria-label'] ?? ''}
                 className={props.menuButton?.class ?? ''}
                 onClick={handleMenuButtonClick}
-                onMouseDown={handleMenuButtonMouseDown}
+                onMouseDown={handleDownEvent}
+                onKeyDown={handleDownEvent}
```

Comment:
What are these changes for? It seems like we would want the focus-visible class to be applied when navigating the menu with the keyboard